timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -523,6 +523,7 @@ void declare_bound_potential(py::module &m) {
             }),
             py::arg("potential"),
             py::arg("params"))
+        .def("get_potential", [](const timemachine::BoundPotential &bp) { return bp.potential; })
```

Comment:
A more explicit name might be `get_unbound_impl`. 

Cons: Would no longer be consistent with argument name `potential`.

Pros: Unambiguous whether it returns `harmonic_bond` potential or its `unbound_impl` in test https://github.com/proteneer/timemachine/blob/e01c188e6dc45df45816350e6b8215937611a127/tests/test_potentials.py#L27-L31

Reply:
Good point about the potential ambiguity. I had considered `get_unbound_impl`, but was thinking that might also be confusing because (as far as I know) the un/bound language is currently only used on the Python side